packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Uncategorized
+
+- Please hold off on new releases of this package until the yarn resolution for `@metamask/providers` is removed.
+  - This is blocked by a `@metamask/snaps-sdk` release with `@metamask/providers` bumped to `>=17.1.1`.
+    - See: [Fix regressions introduced by @metamask/providers@17.1.1](https://github.com/MetaMask/snaps/pull/2579)
+  - Build error fixed by yarn resolution: [MetaMask/core/actions/runs/10011688901/job/27675682526?pr=3645](https://github.com/MetaMask/core/actions/runs/10011688901/job/27675682526?pr=3645)
+
```

Comment:
The packages affected by the yarn resolution are `{accounts,chain,profile-sync}-controller`. It's safe to temporarily merge the yarn resolution into main, as long as these packages aren't released (and even if they are `@metamask/providers` is not a direct dependency of any of these packages).

In any case, these three packages will be excluded from the core releases that follow this PR, since they are blocked by a `snaps-controllers` release as well.